src/settings-ui/Settings.UI/PowerToys.Settings.csproj

```diff
@@ -57,6 +58,9 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="6.0.0" />
+    <PackageReference Include="Microsoft.VariantAssignment.Client" Version="2.4.17140001" />
+    <PackageReference Include="Microsoft.VariantAssignment.Contract" Version="3.0.16990001" />
```

Comment:
where do we get these from and do we need to stub stuff out like we did with telemetry?

Reply:
@BenJKuhn as FYI, these could be a tricky one

Reply:
On Telemetry, we're only moving some files around. This looks more like it'll need a whole project that uses these 3 packages, with everything stubbed out.
@chenss3 , can you please create a project instead that uses these APIs and that project would have to be called here.
What dll files would be needed at runtime for the APIs to work? That's important since those would have to be conditionally added to the installer as well.